src/index.ts

```diff
@@ -380,6 +327,85 @@ export class GalleryComponent extends BaseComponent {
     return null;
   }
 
+  private _escapeHtml = (text: string): string => {
+    return String(text)
+      .replace(/&/g, "&amp;") // Escape '&' first to avoid double escaping
+      .replace(/</g, "&lt;")  // Escape '<'
+      .replace(/>/g, "&gt;")  // Escape '>'
+      .replace(/"/g, "&quot;") // Escape '"'
+      .replace(/'/g, "&#039;"); // Escape "'"
+  };
+
+  private _galleryThumbsTemplate = (thumb): string => {

```

Comment:
While we're talking about types, we could do this too:

```suggestion
  private _galleryThumbsTemplate = (thumb: Thumb): string => {

```